projects/packages/forms/src/dashboard/inbox/export-responses/google-drive.js

```diff
@@ -16,6 +17,10 @@ const GoogleDriveExport = ( { onExport } ) => {
 	const [ isConnected, setIsConnected ] = useState( config( 'gdriveConnection' ) );
 	const { tracks } = useAnalytics();
 
+	const { isUserConnected, handleConnectUser, userIsConnecting, isOfflineMode } = useConnection( {
```

Comment:
I wasn't able to test this redirect. Mostly because it asked me to use the magic link. And when I used it it didn't return me back to the wp-admin as expected.

Reply:
Magic links wouldn't work with a8c accounts, or did you test with non-a8c account?